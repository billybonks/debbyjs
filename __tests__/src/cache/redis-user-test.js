const describeClass = require('../../helpers/class');

describeClass(() => {
  require('./shared/test-for-redis-cache');
});
