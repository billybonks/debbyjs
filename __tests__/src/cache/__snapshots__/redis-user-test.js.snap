// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Unit | RedisUser #fetch it calls redis.get 1`] = `
Array [
  "billybonks-15",
]
`;

exports[`Unit | RedisUser #generateRedisKey returns expected value 1`] = `"billybonks-22"`;

exports[`Unit | RedisUser #save it calls redis.set 1`] = `
Array [
  "billybonks-12",
  "{\\"name\\":\\"bob\\",\\"something\\":\\"awesome\\"}",
  "EX",
  600,
]
`;
